### ISSUE TITLE
Rotate AWS Keys - March 31, 2024

### ISSUE THREAD

**Issue by loripusey** [department-of-veterans-affairs/va.gov-team#76340]
## Summary
We need to rotate the IAM keys in AWS every 60 days. 

## Resources
[Documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/engineering/rotating-iam-keys.md)
[Knowledge sharing meeting recording](https://drive.google.com/file/d/1owR-Pec3urKlulr3ysarHXER2bvbd7Ia/view?ts=65baa17b)

## Tasks
- Rotate the keys

**Comment by benbrasso-agile6** [comment]
@wullaski thiss ready to close?

**Comment by wullaski** [comment]
@benbrasso-agile6 The keys have been rotated and the old ones deactivated. I still have to delete the keys. I'm trying to get access to aws again. I renewed my security key and well now it's not working 😭

**Comment by loripusey** [comment]
@wullaski I'm going to create a new ticket to delete the keys and close this one; we are being pushed to not rollover tickets, so let's break them down as we need to; I'll add the new ticket to the next sprint and put blocked on it, when you get access let me know and I'll add it to a sprint

**Comment by wullaski** [comment]
@loripusey sorry for the runaround. I just got my access key to work and deleted the old keys so we wont need that ticket.

**Comment by loripusey** [comment]
thanks @wullaski I'll delete the new ticket then
